dashboard/app/models/user.rb

```diff
@@ -2185,6 +2185,12 @@ def record_soft_delete
     )
   end
 
+  def donor_school
+    if last_complete_school_info&.school
+      DonorSchool.find_by(nces_id: last_complete_school_info.school.id)
+    end
+  end
```

Comment:
Longer term, the donor name will be important.  See the discussion at https://github.com/code-dot-org/code-dot-org/pull/30591#discussion_r322949623.  Can you use [`donor_teacher_banner_name`](https://github.com/code-dot-org/code-dot-org/blob/6cf1ef9b934d7a089b0e337020f8a1528d99a1c4/dashboard/app/models/user.rb#L1977) now that it's merged?